pallets/pools/src/lib.rs

```diff
@@ -391,8 +427,12 @@ pub mod pallet {
 
 	#[pallet::storage]
 	#[pallet::getter(fn scheduled_update)]
-	pub type ScheduledUpdate<T: Config> =
-		StorageMap<_, Blake2_128Concat, T::PoolId, ScheduledUpdateDetails<T::InterestRate>>;
+	pub type ScheduledUpdate<T: Config> = StorageMap<
+		_,
+		Blake2_128Concat,
+		T::PoolId,
+		ScheduledUpdateDetails<T::InterestRate, T::MaxTokenNameLength, T::MaxTokenSymbolLength>,
```

Comment:
Technically this requires a migration, though I believe as long as we coordinate with the active pool on Altair to ensure we don't have a scheduled pool update when we run the runtime upgrade, we can get away with it (as this map will be empty) /cc @mustermeiszer does that sound right to you?

Reply:
Yes, I think it is reasonable to do it that way. 👍